### ISSUE TITLE
Some NIST RPT documents do not have docnumbers assigned

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-nist#59]
```xml
      <doi type="report-paper_title">10.6028/NBS.RPT.ADHOC</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">92684662</crm-item>
      <crm-item name="book-id" type="number">2515005</crm-item>
      <crm-item name="deposit-timestamp" type="number">201708311440</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T09:21:39Z</crm-item>
      <crm-item name="created" type="date">2017-08-31T18:43:28Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata>
               <contributors>
                  <person_name sequence="first" contributor_role="author">
                     <given_name>Paul R</given_name>
                     <surname>Achbenbach</surname>
                  </person_name>
               </contributors>
               <titles>
                  <title>Application of Mathes heat pumps to Capehart housing at Little Rock Air Force Base:</title>
                  <subtitle>report of Ad Hoc Committee</subtitle>
               </titles>
               <publication_date media_type="online">
                  <year>1959</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS RPT ADHOC</item_number>
               </publisher_item>
               <fr:program xmlns:fr="http://www.crossref.org/fundref.xsd" name="fundref">
                  <fr:assertion name="funder_name">National Institute of Standards and Technology<fr:assertion name="funder_identifier">https://doi.org/10.13039/100000161</fr:assertion>
                  </fr:assertion>
               </fr:program>
               <doi_data>
                  <doi>10.6028/NBS.RPT.ADHOC</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportadhoc.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Apr-Jun1948</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911378</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:00:46Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1948</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Apr-Jun1948</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Apr-Jun1948</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportApr-Jun1948.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Apr-Jun1949</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911379</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:02:20Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1949</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Apr-Jun1949</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Apr-Jun1949</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportApr-Jun1949.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Apr-Jun1950</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911374</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:05:28Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1950</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Apr-Jun1950</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Apr-Jun1950</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportApr-Jun1950.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Apr-Jun1951</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911369</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:04:06Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1951</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Apr-Jun1951</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Apr-Jun1951</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportApr-Jun1951.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.div9</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911380</crm-item>
      <crm-item name="book-id" type="number">2207586</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:00:32Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Mineral Products Division</organization>
               </contributors>
               <titles>
                  <title>National Bureau of Standards Report :</title>
                  <subtitle>Mineral Products Division</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1959</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; div9</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.div9</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportdiv9.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Jan-Jun1971</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911367</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:00:18Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a semi-annual report January through June 1971</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1971</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Jan-Jun1971</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Jan-Jun1971</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportJan-Jun1971.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Jan-Mar1948</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911377</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:02:19Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1948</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Jan-Mar1948</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Jan-Mar1948</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportJan-Mar1948.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Jan-Mar1950</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911368</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T09:58:49Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1950</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Jan-Mar1950</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Jan-Mar1950</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportJan-Mar1950.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Jul-Dec1947</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911376</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:00:45Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1947</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Jul-Dec1947</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Jul-Dec1947</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportJul-Dec1947.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Jul-Sep1949</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911370</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T09:58:50Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1949</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Jul-Sep1949</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Jul-Sep1949</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportJul-Sep1949.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Jul-Sep1950</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911373</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:00:44Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1950</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Jul-Sep1950</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Jul-Sep1950</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportJul-Sep1950.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Oct-Dec1949</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911371</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T09:59:42Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1949</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Oct-Dec1949</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Oct-Dec1949</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportOct-Dec1949.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
   <query key="RPT">
      <doi type="report-paper_title">10.6028/NBS.RPT.Oct-Dec1950</doi>
      <crm-item name="publisher-name" type="string">National Institute of Standards and Technology (NIST)</crm-item>
      <crm-item name="prefix-name" type="string">National Institute of Standards and Technology</crm-item>
      <crm-item name="member-id" type="number">4068</crm-item>
      <crm-item name="citation-id" type="number">85911372</crm-item>
      <crm-item name="book-id" type="number">2056501</crm-item>
      <crm-item name="deposit-timestamp" type="number">201610201500</crm-item>
      <crm-item name="owner-prefix" type="string">10.6028</crm-item>
      <crm-item name="last-update" type="date">2018-03-06T10:02:18Z</crm-item>
      <crm-item name="created" type="date">2016-10-20T19:40:51Z</crm-item>
      <crm-item name="citedby-count" type="number">0</crm-item>
      <doi_record>
         <report-paper>
            <report-paper_metadata language="en">
               <contributors>
                  <organization sequence="first" contributor_role="author">Applied Mathematics Division</organization>
               </contributors>
               <titles>
                  <title>Projects and publications of the Applied Mathematics Division :</title>
                  <subtitle>a quarterly report</subtitle>
               </titles>
               <edition_number>0</edition_number>
               <publication_date media_type="online">
                  <year>1950</year>
               </publication_date>
               <publisher>
                  <publisher_name>National Bureau of Standards</publisher_name>
                  <publisher_place>Gaithersburg, MD</publisher_place>
               </publisher>
               <institution>
                  <institution_name>National Bureau of Standards</institution_name>
                  <institution_acronym>NBS</institution_acronym>
                  <institution_place>Gaithersburg, MD</institution_place>
               </institution>
               <publisher_item>
                  <item_number item_number_type="report-number">NBS report ; Oct-Dec1950</item_number>
               </publisher_item>
               <doi_data>
                  <doi>10.6028/NBS.RPT.Oct-Dec1950</doi>
                  <resource>https://nvlpubs.nist.gov/nistpubs/Legacy/RPT/nbsreportOct-Dec1950.pdf</resource>
               </doi_data>
            </report-paper_metadata>
         </report-paper>
      </doi_record>
   </query>
```

**Comment by ronaldtse** [comment]
These are items that require updating the PubID scheme. We have to bring this to the attention of the NIST Library.

**Comment by ronaldtse** [comment]
@andrew2net let's just parse the date portion as the docnumber for now.

**Comment by andrew2net** [comment]
fixed in v 1.9.8
